persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -16,8 +16,15 @@ CREATE TABLE Conversation (
     muted_status TEXT AS ConversationEntity.MutedStatus DEFAULT "ALL_ALLOWED" NOT NULL,
     muted_time INTEGER DEFAULT 0 NOT NULL,
     creator_id TEXT NOT NULL,
+    -- when even a conversation event is recived, last_modified_date is updated for spicific events
```

Comment:
not any conversation event, but only events that affect conversationsList sorting.  Like message event should not update that sorting, so `last_modified_date` stays the same in that case

Reply:
@borichellow pls commit suggestion